doc/build-unix.md

```diff
@@ -308,3 +308,32 @@ To build executables for ARM:
 
 
 For further documentation on the depends system see [README.md](../depends/README.md) in the depends directory.
+
+Building on FreeBSD
+--------------------
+
+(Updated as of FreeBSD 10.3)
+
+Clang is installed by default as `cc` compiler, this makes it easier to get
+started than on OpenBSD. Installing dependencies:
+
+    pkg install autoconf automake libtool pkgconf
+    pkg install boost-libs openssl libevent2
+
+(`libressl` instead of `openssl` will also work)
+
+For the wallet (optional):
+
+    pkg install db5
+
+This will give a warning "configure: WARNING: Found Berkeley DB other
+than 4.8; wallets opened by this build will not be portable!", but as FreeBSD never
+had a binary release, this may not matter. If backwards compatibility
+with 4.8-built Bitcoin Core is needed follow the steps under "Berkeley DB" above.
```

Comment:
I'm not sure that "this may not matter" is the correct wording here. Additionally, might want to make the warning in the next sentence more clear. "If you need to open wallets with Bitcoin Core and wish to have the option to open them with binary releases or releases on different systems in the future, you may need to follow the steps under "Berkeley DB" above".